src/placeholder/parameter/parameter.py

```diff
@@ -250,6 +273,8 @@ def consolidate(cls, instance_list, exclude=None):
 
     Parameters
     ----------
+    cls : Object
```

Comment:
I think we might need to be clear here the class has to be a `dataclass`, otherwise I am not sure if `fields` will work :)

Reply:
dbc9397 :)